models/project/board.go

```diff
@@ -224,6 +227,29 @@ func GetBoards(ctx context.Context, projectID int64) (BoardList, error) {
 	return append([]*Board{defaultB}, boards...), nil
 }
 
+func GetBoardsAndCount(ctx context.Context, projectID int64) (BoardList, int64, error) {
+	engine := db.GetEngine(ctx)
+	boards := make([]*Board, 0, setting.UI.IssuePagingNum)
```

Comment:
```suggestion
	boards := make([]*Board, 0, 10)
```
we use some number between 5-10 depending on what we thing is most common
if you like to introduce pagination you should do so by adding an option struct

Reply:
we migt should refactor that and use a const ...